### ISSUE TITLE
some apbss files not published

### ISSUE THREAD

**Issue by prjemian** [BCDA-APS/apstools#347]
These are the only `apsbss` files packaged now:

```
(bluesky_2020_5) mintadmin@mint-vm:~/.../projects/apstools$ tar tjf /tmp/conda-bld/noarch/apstools-1.3.0rc1-py_0.tar.bz2 | grep apsbss
site-packages/apstools/beamtime/apsbss_makedb.py
site-packages/apstools/beamtime/apsbss_ophyd.py
site-packages/apstools/beamtime/apsbss.py
```

There are more.

**Comment by prjemian** [comment]
fixed in fc84fa0

**Comment by prjemian** [comment]
One more: `apsbss_ioc.sh`

**Comment by prjemian** [comment]
Consult: https://setuptools.readthedocs.io/en/latest/pkg_resources.html#resourcemanager-api

**Comment by prjemian** [comment]
Nope, not that.  Here: https://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files

**Comment by prjemian** [comment]
fixed in 1.3.0rc3
